dataedit/views.py

```diff
@@ -758,6 +759,26 @@ def view_delete(request, schema, table):
     return redirect("/dataedit/view/" + schema + "/" + table)
 
 
+def create_graph(request, schema, table):
+
+    if request.method == 'POST':
+        print(request.POST, schema, table)
+        # TODO save an instance of tablegraph here
+        return redirect(
+            "/dataedit/view/{schema}/{table}".format(schema=schema, table=table)
+        )
+    else:
+        # TODO get the columns from the schema and the table
```

Comment:
@MGlauer , @gnn do you know a function I could use for that?

Reply:
Is it necessary to do this server-side? The table functions query all columns via the API and [process their name and datatype](https://github.com/OpenEnergyPlatform/oeplatform/blob/3440f5e5b89a31d366fd2e99115bc92c44d4f930/dataedit/static/dataedit/backend.js#L231). Could you use this instead?

Reply:
What I need to have is the list of columns of the table so that the dropdown for x and y are populated. If I want to use forms and not write js code in the template I need to have this list here, but the list could be provided via the GET.

I will describe my idea for the model of TableGraph in the PR's description

Reply:
@MGlauer, I described it